noir_stdlib/src/meta/op.nr

```diff
@@ -39,7 +39,8 @@ impl UnaryOp {
         } else if self.is_dereference() {
             quote { * }
         } else {
-            crate::mem::zeroed()
+            let op = self;
+            crate::panic::panic(f"Unexpected unary operator in UnaryOp::quoted: {op}")
```

Comment:
Does interpolating `self` not work? Can you create an issue for that if so?

Reply:
Ah, I initially wrote `{self.op}` 🤦 . I think `{self}` will work fine.